test/wh_test_crypto.c

```diff
@@ -653,11 +731,19 @@ int whTest_CryptoServerConfig(whServerConfig* config)
         return WH_ERROR_BADARGS;
     }
 
+    /* set cancelSeqP */
+    cancelSeqP = &server->cancelSeq;
```

Comment:
extremely unlikely, but technically this could be accessed by the client before it is set by the server. Not gonna happen so don't really care, but further case for testing this functionality sequentially, vs in a threaded environment.

Reply:
It's fine, this won't happen when custom servers is on due to `serverDelay` and otherwise cancel late will kick in and the test will still pass